config/application.rb

```diff
@@ -7,7 +7,12 @@
 require 'active_record/railtie'
 require 'active_storage/engine'
 require 'action_controller/railtie'
+# require "action_mailer/railtie"
+# require 'action_mailbox/engine'
+# require 'action_text/engine'
 require 'action_view/railtie'
+# require "action_cable/engine"
+# require "sprockets/railtie"
```

Comment:
I wouldn't add these commented lines, seems unnecessary, since we will never use these components.